compiler/test/dotty/tools/repl/ReplTest.scala

```diff
@@ -23,6 +23,9 @@ class ReplTest private (out: ByteArrayOutputStream) extends ReplDriver(
     output
   }
 
+  /** Get the stored output as list from `out`, resetting the buffer */
+   def storedOutputAsList() : List[String] = storedOutput().trim().split("\r?\n").toList
```

Comment:
I would move this into `ReplCompilerTests.scala`.

Also, what about?
```scala
def lines() = storedOutput().lines.toList
```

Reply:
> I would move this into `ReplCompilerTests.scala`.
I thought it might be used in other derived classes, but OK, will move.
> 
> Also, what about?
> 
> ```scala
> def lines() = storedOutput().lines.toList
> ```
Thanks for hint!
It's not exactly the solution but this one works:
```scala
def lines() = storedOutput().trim().lines.toList
```
I will update commit in a few minutes